patches/spec.fetched.json/01-otps-case-correction.patch

```diff
@@ -2,7 +2,7 @@ diff --git a/spec/oas3.patched/openapi/public/openapi3.yaml b/spec/oas3.patched/
 index 6ae109f..dc2792f 100644
 --- a/spec/oas3.patched/openapi/public/openapi3.yaml
 +++ b/spec/oas3.patched/openapi/public/openapi3.yaml
```

Comment:
Should this be replaced/updated to patch the following files:
- paths/user/otp/recovery-codes.yaml
- paths/user/otp/sms/receive.yaml
- paths/user/otp/verify/otp.yaml

Reply:
I believe it is Github highlights and not version update, as you can see here https://github.com/equinix-labs/metal-java/blob/main/patches/spec.fetched.json/01-otps-case-correction.patch#L3 those lines in red still exists

Reply:
I didn't see the other files in the patch. I saw openapi.yaml and thought this patch was being applied against the previous incarnation of the file (all in one file).